readthedocs/builds/models.py

```diff
@@ -466,6 +467,13 @@ def finished(self):
         """Return if build has a finished state."""
         return self.state == BUILD_STATE_FINISHED
 
+    @property
+    def raw_log(self):
+        raw_log = loader.get_template('restapi/log.txt').render(
```

Comment:
Use the shortcut `render_to_string`, https://docs.djangoproject.com/en/2.0/topics/templates/#django.template.loader.render_to_string